assets/src/containers/Course.js

```diff
@@ -83,66 +83,50 @@ function Course (props) {
                   />
                 </Card>
               ) : (
-                <>
-                  <Route
-                    path='/courses/:courseId/'
-                    exact
-                    render={props =>
-                      <IndexPage
-                        {...props}
-                        courseInfo={courseInfo}
-                        courseId={courseId}
-                        enrollmentTypes={enrollmentTypes}
-                        isAdmin={user.admin}
-                      />}
-                  />
-                  <Route
-                    path='/courses/:courseId/grades'
-                    render={props =>
-                      <GradeDistribution
-                        {...props}
-                        disabled={!courseInfo.course_view_options.gd}
-                        courseId={courseId}
-                        user={user}
-                        enrollmentTypes={enrollmentTypes}
-                        isAdmin={user.admin}
-                      />}
-                  />
-                  <Route
-                    path='/courses/:courseId/assignmentsv1'
-                    render={props =>
-                      <AssignmentPlanning
-                        {...props}
-                        disabled={!courseInfo.course_view_options.apv1}
-                        courseId={courseId}
-                        enrollmentTypes={enrollmentTypes}
-                        isAdmin={user.admin}
-                      />}
-                  />
-                  <Route
-                    path='/courses/:courseId/assignments'
-                    render={props =>
-                      <AssignmentPlanningV2
-                        {...props}
-                        disabled={!courseInfo.course_view_options.ap}
-                        courseId={courseId}
-                        enrollmentTypes={enrollmentTypes}
-                        isAdmin={user.admin}
-                      />}
-                  />
-                  <Route
-                    path='/courses/:courseId/resources'
-                    render={props =>
-                      <ResourcesAccessed
-                        {...props}
-                        disabled={!courseInfo.course_view_options.ra}
-                        courseInfo={courseInfo}
-                        courseId={courseId}
-                        enrollmentTypes={enrollmentTypes}
-                        isAdmin={user.admin}
-                      />}
-                  />
-                </>
+                <Switch>
+                  <Route path='/courses/:courseId/' exact>
```

Comment:
For all the routes below, I see `{...props}` is removed. was that redundant? or props are available in some other way?

I know there is one use case that instructor given a choice to select what views they want for the course for example disabling grade distribution, I think locally we are maintaining the state using props if he doesn't launch the tool. 

I haven't tested the UI still, I will in a bit.

Reply:
This change doesn't seem to have side effect and seems to be safe